client/src/components/utils/sasCode.ts

```diff
@@ -53,17 +53,17 @@ export function wrapCodeWithOutputHtml(code: string): string {
   const activeProfile = profileConfig.getActiveProfileDetail();
   const outputDestination =
     activeProfile &&
-    activeProfile.profile.connectionType !== ConnectionType.Rest
+      activeProfile.profile.connectionType !== ConnectionType.Rest
```

Comment:
the build is failing due to format check. Have you updated dependencies (Prettier version in particular) and npm run format on this? looks like this line of change is not needed

Reply:
fixed now.